blog/2022-10-18-rename-imperial-unit-system.md

```diff
@@ -0,0 +1,20 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "Rename Imperial unit system to US Customary"
```

Comment:
I have created a separate blog post, but if you feel it should be merged into the previous blog post I can adjust accordingly.

Reply:
I think it should be a single blog post, since the two changes are closely related 👍

Reply:
done